### PR TITLE
qt-creator: version bumped to 4.0.3.

### PR DESCRIPTION
**Pull request by Florin65**

### CODE REVIEW

File: qt5-apps/qt-creator/DETAILS [hunk 1]
```diff
@@ -1,18 +1,18 @@
           MODULE=qt-creator
-         VERSION=opensource-src-4.0.2
+            VERS=4.0.3
+         VERSION=opensource-src-$VERS
           SOURCE=$MODULE-$VERSION.tar.gz
-      SOURCE_URL=http://download.qt-project.org/official_releases/qtcreator/4.0/4.0.2/
-      SOURCE_VFY=sha256:c2fce74fef0cc3a8f5c03d7ed1e95ccfe508d855ee750647896876824fcaee10
+      SOURCE_URL=http://download.qt-project.org/official_releases/qtcreator/${VERS%.*}/$VERS/
+      SOURCE_VFY=sha256:
```
Comment (stumbles) [inline qt5-apps/qt-creator/DETAILS]: Need a sha256;

File: qt5-apps/qt-creator/DETAILS [hunk 2]
```diff
@@ -1,18 +1,18 @@
           MODULE=qt-creator
-         VERSION=opensource-src-4.0.2
+            VERS=4.0.3
+         VERSION=opensource-src-$VERS
```
Comment (stumbles) [inline qt5-apps/qt-creator/DETAILS]: This looks better to me;

          MODULE=qt-creator
         VERSION=4.0.3
          SOURCE=$MODULE-opensource-src-$VERSION.tar.gz
      SOURCE_URL=http://download.qt-project.org/official_releases/qtcreator/${VERSION%.*}/$VERSION/

File: qt5-apps/qt-creator/DEPENDS [hunk 3]
```diff
@@ -1 +1,9 @@
-depends  qt5
+depends qt5
+depends llvm
```
Comment (stumbles) [inline qt5-apps/qt-creator/DEPENDS]: AFAIK and not heard any chatter from kde/qt folks, llvm is not a hard depends. qtcreator automatically detects gcc when launched and any other compilers you have installed require manually adding via Tools -> Options. Will these changes make llvm the default?

**Comment by Florin65** [comment]
we can put it optional depend, np.

**Pull request merged**